Cargo.toml

```diff
@@ -269,7 +269,7 @@ smallvec = { version = "1.13.1", features = [
   "union",
   "const_new",
 ] }
-sourcemap = "6.0.2"
+sourcemap = "6.4.1"
```

Comment:
Lock file need to be updated

Reply:
no, https://github.com/vercel/turbo/blob/main/Cargo.lock#L7744-L7745 there's semver-bumped version already in lockfile and it is the only version so changing in here doesn't trigger lockfile update.